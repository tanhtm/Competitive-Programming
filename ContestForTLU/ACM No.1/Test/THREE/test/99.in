732183593000000000
