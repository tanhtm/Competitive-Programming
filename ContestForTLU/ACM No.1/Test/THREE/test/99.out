0 1 3 5 6 10 13 19 21 25 26 27 28 32 34 36 37 
12 14 15 16 29 30 38 
